11KateM/HRLibrary.UnitTests/ShowUnitTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using HRLibrary;
+using NUnit.Framework;
+namespace HRLibrary.UnitTests
+{
+    [TestFixture]
+    public class ShowUnitTests
+    {
+
+        private Show CreateTestMedicine()
```

Comment:
Странное название метода. При чем тут лекарство (medicine)? Лучше бы так
```suggestion
        private Show CreateTestShow()
```